ArcGISRuntimeSDKQt_CppSamples/Geometry/Buffer/README.md

```diff
@@ -0,0 +1,19 @@
+# Buffer
+This sample demonstrates how to use the GeometryEngine::Buffer to generate a polygon of a fixed distance around a point.
```

Comment:
Should `GeometryEngine::Buffer` be `GeometryEngine::buffer method`?  Not sure what we do in other places.